Source/WebKit/_WebKit_SwiftUI/API/View+WebViewModifiers.swift

```diff
@@ -30,50 +30,56 @@ extension View {
     @available(WK_IOS_TBA, WK_MAC_TBA, WK_XROS_TBA, *)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
-    public func webViewBackForwardNavigationGestures(_ value: WebView.BackForwardNavigationGesturesBehavior = .automatic) -> some View {
+    public nonisolated func webViewBackForwardNavigationGestures(_ value: WebView.BackForwardNavigationGesturesBehavior) -> some View {
```

Comment:
Can you give more explanation about the two changes here? I’m assuming we added `nonisolated` as a general alignment with SwiftUI’s view modifiers, but why drop the default value? Just because it’s effectively inert without specifying something else?

Reply:
> I’m assuming we added nonisolated as a general alignment with SwiftUI’s view modifiers

Yup, that's correct.

>  Just because it’s effectively inert without specifying something else?

Yup! And also fits in with the general SwiftUI view modifier pattern of not having automatic be the default value, such as https://developer.apple.com/documentation/swiftui/view/scrollcontentbackground(_:)